packages/app-project/src/screens/ClassifyPage/components/RecentSubjects/components/SubjectPreview/components/FavouritesButton/FavouritesButton.js

```diff
@@ -0,0 +1,44 @@
+import counterpart from 'counterpart'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import MetaToolsButton from '../MetaToolsButton'
+import HeartIcon from './HeartIcon'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+export const Favourite = styled(HeartIcon)`
+  fill: ${props => props.filled ? props.theme.global.colors.statusColors.error : 'none'};
+`
+
+export default function FavouritesButton (props) {
+  const { className, checked, disabled, onClick } = props
+  const label = checked ? 'FavouritesButton.remove' : 'FavouritesButton.add'
+  const filled = checked ? 'true' : undefined
+  return (
+    <MetaToolsButton
+      aria-checked={checked}
+      disabled={disabled}
+      icon={<Favourite className={className} color='dark-5' filled={filled} size='1em' />}
```

Comment:
You don't need `className` here; if you log it, it's empty (both checked and unchecked), and there aren't any warnings thrown by it not being there.

If you're starting with a grommet component, [it appears to be handled for you](https://github.com/grommet/grommet-icons/blob/master/src/js/StyledIcon.js#L40)

Reply:
I'll try this but I think you are supposed to pass `className` down when you use `styled()`. `Favourite` here is a `styled(HeartIcon)`.